cfme/configure/access_control.py

```diff
@@ -256,7 +256,10 @@ def description(self):
 class Group(Updateable, Pretty):
     group_form = Form(
         fields=[
-            ('ldap_groups_for_user', Select("//*[@id='ldap_groups_user']")),
+            ('ldap_groups_for_user', {
+                version.LOWEST: Select("//*[@id='ldap_groups_user']"),
+                "5.5": AngularSelect("ldap_groups_user")}
```

Comment:
Just replace this with the Angular, we don't need to support anything under 5.6 now